packages/frontend-plugin-api/src/analytics/AnalyticsContext.tsx

```diff
@@ -37,7 +37,7 @@ export const useAnalyticsContext = (): AnalyticsContextValue => {
   // Provide a default value if no value exists.
   if (theContext === undefined) {
     return {
-      pluginId: 'root',
+      pluginId: 'app',
```

Comment:
@iamEAP is there any risk in changing this id?

Reply:
No, I don't think so.  It might be worth documenting (changeset and maybe even somewhere in docs?), but there's no risk.  If anything, it makes an edge-case much clearer.  ...This (seeing `root` as a plugin in an analytics service) most often comes up in situations where pages are added to a Backstage App that aren't tied to a plugin (e.g. a custom route in an app).  ...And `app` makes much more sense in that case.